tests/cypress/integration/z-newfold-labs/wp-module-onboarding/1-initial-steps/TC_02-exit-to-wordpress.cy.js

```diff
@@ -33,7 +31,7 @@ describe( 'Exit to WordPress', function () {
 	it( 'Exit to WordPress Page', () => {
 		cy.get( '.nfd-onboarding-etw__trigger' ).click();
 		cy.get( '.nfd-onboarding-etw__buttons > .is-primary' ).click();
-		cy.url( { timeout: 12000 } ).should( 'contain', 'bluehost' );
+		cy.url( { timeout: 12000 } ).should( 'contain', 'index.php' );
```

Comment:
This has to be `bluehost` otherwise it means that our redirect does not work.

Reply:
This isn't going to Bluehost dashboard, this is redirecting to WP dashboard.

Reply:
That means the test is failing. Can we find out why?